_stbt/utils.py

```diff
@@ -88,8 +89,51 @@ def find_import_name(filename):
     return import_dir, import_name
 
 
+if sys.version_info.major == 2:  # Python 2
+    text_type = unicode  # pylint: disable=undefined-variable
```

Comment:
python-future already has this as `future.utils.text_type` (it's the native `unicode` type on Python 2, not `newstr`).